src/test/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/JtsLayerTest.java

```diff
@@ -31,7 +42,7 @@ public void testLayerCollection() {
         String layerName = "Points of Interest";
         List<Geometry> geometries = new ArrayList<>();
 
-        JtsLayer layer = new JtsLayer(layerName, geometries);
+        JtsLayer layer = new JtsLayer(layerName, geometries, Extent.DEFAULT);
```

Comment:
Don't need to modify all the tests with a 3-arg if you remove the breaking API change to the constructors to the layer.